WTWSMS/history/titles/c_alexandretta.txt

```diff
@@ -55,6 +55,13 @@
 500.1.1 = {
 	holder = 6889
 }
-540.1.1={
-	holder=122503 # Celeus
+524.1.1 = {
+    holder = 1310010
+    liege = "k_cilicia"
+}
+527.1.1 = {
+	liege = "k_cilicia"
+}
+540.1.1 = {
+	holder = 122503 # Celeus
```

Comment:
After looking at this county more in detail, it seems to be attached to Oriens through history.

Reply:
You mean changing the liege?

Reply:
No, I mean the county history itself. You are attaching it to Cilicia, currently it is a part of Oriens.

Reply:
Returned back to Oriens then.